client/components/recurring-payment-icon/index.js

```diff
@@ -0,0 +1,64 @@
+import { __ } from '@wordpress/i18n';
+import React, { useState } from 'react';
+import styled from '@emotion/styled';
+import { Popover as PopoverComponent } from '@wordpress/components';
+import icon from './icon.svg';
+
+const Icon = styled.img`
+	height: 12px;
+	width: 12px;
```

Comment:
Can we make these the same size as the text? At present, I think the text size is `14px` by default.

Reply:
Done in [97d80d7](https://github.com/woocommerce/woocommerce-gateway-stripe/pull/4299/commits/97d80d7a55c7d5f09138a9fcb7bc13a3677b1a6d)